src/components/common/MemosTable/AddReply/Reply.css

```diff
@@ -1,71 +1,68 @@
-
 .popup {
-    position: fixed;
-    top: 0;
-    left: 0;
-    width: 100%;
-    height: 100%;
-    background-color: rgba(0, 0, 0, 0.2);
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    overflow: hidden;
-    z-index: 100;
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.2);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  overflow: hidden;
+  z-index: 100;
 }
 .popup-inner {
-    position: relative;
-    padding: 32px;
-    width: 100%;
-    max-width: 640px;
-    background-color: rgb(61, 61, 68);
-    color: rgb(54, 50, 50);
+  position: relative;
+  padding: 32px;
+  width: 100%;
+  max-width: 640px;
+  background-color: rgb(61, 61, 68);
+  color: rgb(54, 50, 50);
 }
 .popup-inner .close-btn {
-    position: absolute;
-    top: 16px;
-    right: 16px;
-    background-color: rgba(43, 114, 190, 0.509);
+  position: absolute;
+  top: 16px;
+  right: 16px;
+  background-color: rgba(43, 114, 190, 0.509);
 }
 
 .popup-inner h1 {
-    color: white;
-    display: flex;
-    justify-content: center;
-    font-size: 1.5rem;
+  color: white;
+  display: flex;
+  justify-content: center;
+  font-size: 1.5rem;
 }
 
-.close-btn h1{
-    color: white;
```

Comment:
All of this cleaned up formatting should definitely help make the codebase easier to read and navigate.